fix: add sbo capabilities (#83)
redhat-developer/app-services-guides: code-examples/quarkus-kafka-quickstart/README.md

Quarkus Kafka Quickstart
========================

> 

Reviewer: ```suggestion
To see how to use this code example follow the [quick start](../../kafka-quarkus-dev).

## Additional information on this code example

```

> Based on https://github.com/quarkusio/quarkus-quickstarts/blob/main/kafka-quickstart/README.md

Reviewer: ```suggestion
This code example is based the [code example from Quarkus](https://github.com/quarkusio/quarkus-quickstarts/blob/main/kafka-quickstart/README.md)
```

This project illustrates how you can interact with Apache Kafka using MicroProfile Reactive Messaging.

> ## RHOAS addons

Reviewer: ```suggestion
### Service Binding
```

> ## RHOAS addons

Reviewer: Everything below here should be in the quick start. We don't need it hanging out in this random other file.
